.gitlab-ci.yml

```diff
@@ -624,6 +624,32 @@ check-try-runtime:
     # Check that everything compiles with `try-runtime` feature flag.
     - cargo check --features try-runtime --all
 
+# More info can be found here: https://github.com/paritytech/polkadot/pull/5865
+# Works only in PRs
+check-runtime-migration:
+  stage:                           stage3
+  # this is an artificial job dependency, for pipeline optimization using GitLab's DAGs
+  needs:
+    - job:                         test-node-metrics
+      artifacts:                   false
+  <<:                              *test-pr-refs
+  <<:                              *docker-env
+  <<:                              *compiler-info
+  script:
+    - |
+      export has_runtimemigration_label=$(curl -sS -H "Accept: application/vnd.github+json" \
+        -H "Authorization: token $GITHUB_PR_TOKEN" \
+        https://api.github.com/repos/paritytech/polkadot/issues/$CI_COMMIT_REF_NAME/labels | grep "runtimemigration" | wc -l)
```

Comment:
perhaps this should use e1 or whatever applicable label code instead of relying on the name not changing, but otherwise lgtm

Reply:
Agree, changed to E1